gen-idea.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 # Check if current directory is the liferay-portal project folder
-PORTAL=$HOME/projects/liferay-portal
+PORTAL=$PWD
```

Comment:
@andreldm, now you can call your alias from the liferay project directory or add this command to your alias: "cd $HOME/projects/liferay-portal".